meson.build

```diff
@@ -476,15 +468,16 @@ uwp_opt = get_option('uwp').require(
 uwp = cc.find_library('windowsapp', required: uwp_opt)
 if uwp.found()
     dependencies += uwp
-    features += 'uwp'
     sources += files('osdep/path-uwp.c')
 endif
+features += {'uwp': uwp.found()}
 
 if win32
     sources += files('osdep/timer-win2.c',
                      'osdep/w32_keyboard.c',
                      'osdep/windows_utils.c')
 endif
+features += {'win32-executable': win32 and get_option('cplayer')}
```

Comment:
waf has `os-win32 || !(!(os-cygwin))` (what?) and a check for `WINDRES` here

Reply:
I defined `win32` in meson as `windows` or `cygwin`. Now, I don't know how windows works but in meson the `windres` thingy is a module which should fail if it's not there.